diff --git a/src/integration.ts b/src/integration.ts
index 780e9ac7ec40aafc1024066fc618fbc5f825935c..1fc083a0173b54a4c17ab79e1c309d379a455b87 100644
--- a/src/integration.ts
+++ b/src/integration.ts
@@ -237,6 +237,7 @@ export function c(size) {
                   }
                   if (id === "\0debug-browser") {
                     return `
+var module = { exports: {} };
 function debug(namespace) {
   function d(...args) {
     if (typeof localStorage !== "undefined" && localStorage.getItem("DEBUG")) {
@@ -250,6 +251,7 @@ debug.disable = function() {};
 debug.enabled = function() { return false; };
 debug.default = debug;
 module.exports = debug;
+export default debug;
 `;
                   }
                 },
